appveyor.yml

```diff
@@ -0,0 +1,47 @@
+# Download script file from GitHub
```

Comment:
Out of curiosity, why appveyor? I thought that gh actions were the goto for new work now, and you already have them wired up here

Reply:
So [this](https://github.com/mrc-ide/squire_js/blob/feat/windows/.github/workflows/e2e.yml) was my attempt at getting it to run on gh actions.

After tinkering with Chrome/Chromedriver [versions](https://github.com/mrc-ide/squire_js/runs/608173190?check_suite_focus=true#step:7:70) for a bit too long, I thought it was more important to simply get our COVID model tested 😆 

The great thing about appveyor is that the workers have all of the browsers and correct drivers installed. When there's a `chrome/install` action, gh actions could compete.

Reply:
seems like a good shout :)